packages/suite/src/views/dashboard/DesktopSuitePromoBanner.tsx

```diff
@@ -161,7 +161,7 @@ const StoreBadge = ({
     );
 };
 
-export const PromoBanner = () => {
+export const DesktopSuitePromoBanner = () => {
```

Comment:
It's confusing to call it just "PromoBanner" since we have two of them:
1. Universal promo banner in Dashboard
2. Desktop suite promo banner in Settings